### PR TITLE
chore: fix compilation after textfield renaming

### PR DESCRIPTION
**Pull request by ZheSun88**
related changes https://github.com/vaadin/flow-components/commit/8b6c3f1050081244ec1393232ff9acbbb031b91e

### CODE REVIEW

File: vaadin-testbench-unit-shared/src/main/java/com/vaadin/flow/component/textfield/TextFieldTester.java [hunk 1]
```diff
@@ -24,7 +24,7 @@
  */
 @Tests({ TextField.class, PasswordField.class, EmailField.class,
         BigDecimalField.class })
-public class TextFieldTester<T extends InternalFieldBase<T, V>, V>
+public class TextFieldTester<T extends TextFieldBase<T, V>, V>
```
Comment (mshabarov) [inline vaadin-testbench-unit-shared/src/main/java/com/vaadin/flow/component/textfield/TextFieldTester.java:27]: class import is missing?

Comment (ZheSun88) [reply vaadin-testbench-unit-shared/src/main/java/com/vaadin/flow/component/textfield/TextFieldTester.java:27]: interesting.. locally it builds for me.. aww.

Comment (ZheSun88) [reply vaadin-testbench-unit-shared/src/main/java/com/vaadin/flow/component/textfield/TextFieldTester.java:27]: oh, it failed on other part.. for this line, it doesn't need extra class import

```
/opt/agent/work/ef9ade89d4c3cefd/vaadin-testbench-unit-junit5/src/test/java/com/vaadin/testbench/unit/ComponentQueryTest.java:[28,43] cannot find symbol
[11:30:05]	[Step 4/5] [ERROR]   symbol:   class InternalFieldBase
[11:30:05]	[Step 4/5] [ERROR]   location: package com.vaadin.flow.component.textfield
[11:30:05]	[Step 4/5] [ERROR] /opt/agent/work/ef9ade89d4c3cefd/vaadin-testbench-unit-junit5/src/test/java/com/vaadin/testbench/unit/ComponentQueryTest.java:[496,19] cannot find symbol
[11:30:05]	[Step 4/5] [ERROR]   symbol:   class InternalFieldBase
[11:30:05]	[Step 4/5] [ERROR]   location: class com.vaadin.testbench.unit.ComponentQueryTest
[11:30:05]	[Step 4/5] [ERROR] -> [Help 1]
```

**Comment by ZheSun88** [comment]
there is another usage for the old `InternalFieldBase`. 
now it has been updated

**Pull request merged**